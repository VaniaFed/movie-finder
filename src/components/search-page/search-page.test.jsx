import { shallow } from 'enzyme';
import { SearchPage } from './index';

describe('<SearchPage />', () => {
    const movies = [
        {
            budget: 44000000,
            genres: ['Crime', 'Drama', 'Mystery', 'Western'],
            id: 273248,
            overview:
                'Bounty hunters seek shelter from a raging blizzard and get caught up in a plot of betrayal and deception.',
            poster_path:
                'https://image.tmdb.org/t/p/w500/fqe8JxDNO8B8QfOGTdjh6sPCdSC.jpg',
            release_date: '2015-12-25',
            revenue: 155760117,
            runtime: 167,
            tagline: 'No one comes up here without a damn good reason.',
            title: 'The Hateful Eight',
            vote_average: 7.6,
            vote_count: 5584
        },
        {
            budget: 44000000,
            genres: ['Crime', 'Drama', 'Mystery', 'Western'],
            id: 273248,
            overview:
                'Bounty hunters seek shelter from a raging blizzard and get caught up in a plot of betrayal and deception.',
            poster_path:
                'https://image.tmdb.org/t/p/w500/fqe8JxDNO8B8QfOGTdjh6sPCdSC.jpg',
            release_date: '2015-12-25',
            revenue: 155760117,
            runtime: 167,
            tagline: 'No one comes up here without a damn good reason.',
            title: 'The Hateful Eight',
            vote_average: 7.6,
            vote_count: 5584
        },
        {
            budget: 44000000,
            genres: ['Crime', 'Drama', 'Mystery', 'Western'],
            id: 273248,
            overview:
                'Bounty hunters seek shelter from a raging blizzard and get caught up in a plot of betrayal and deception.',
            poster_path:
                'https://image.tmdb.org/t/p/w500/fqe8JxDNO8B8QfOGTdjh6sPCdSC.jpg',
            release_date: '2015-12-25',
            revenue: 155760117,
            runtime: 167,
            tagline: 'No one comes up here without a damn good reason.',
            title: 'The Hateful Eight',
            vote_average: 7.6,
            vote_count: 5584
        }
    ];
    const tree = shallow(<SearchPage movies={movies} onSearch={f => f} />);
    it('should be rendered with no movies', () => {
        expect(tree).toMatchSnapshot();
    });
});
