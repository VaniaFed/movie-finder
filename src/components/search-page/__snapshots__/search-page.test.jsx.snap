// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchPage /> should be rendered with no movies 1`] = `ShallowWrapper {}`;
